docs/source/markdown/podman-machine-set.1.md

```diff
@@ -8,17 +8,29 @@ podman\-machine\-set - Sets a virtual machine setting
 
 ## DESCRIPTION
 
-Sets an updatable virtual machine setting.
-
-Options mirror values passed to `podman machine init`. Only a limited
-subset can be changed after machine initialization.
+Change a machine setting.
 
 ## OPTIONS
 
+#### **--cpus**=*number*
+
+Number of CPUs.
+Only supported for QEMU machines.
+
+#### **--disk-size**=*number*
+
+Size of the disk for the guest VM in GB.
+Can only be increased. Only supported for QEMU machines.
+
 #### **--help**
 
 Print usage statement.
 
+#### **--memory**, **-m**=*number*
+
+Memory (in MB).
+Only supported for QEMU machines.
+
 #### **--rootful**=*true|false*
 
 Whether this machine should prefer rootful (`true`) or rootless (`false`)
```

Comment:
I know not part of this change, but it looks like there's an extra space in front of the first word in this line.  Also, the default value is not listed.  Is it true or false, or will the machine not start without this option?

Reply:
Will remove the extra space, thanks!

The default value will be set in `podman machine init`. This is a the `set` command, which changes machine settings after the machine is created. There is no default value; if the flag is not explicitly used, the memory of the machine will not change.